stripe/src/main/java/com/stripe/android/view/CountryCode.kt

```diff
@@ -0,0 +1,20 @@
+package com.stripe.android.view
+
+import androidx.annotation.Size
+import java.util.Locale
+
+internal data class CountryCode(
+    @Size(2) private val value: String,
+) {
```

Comment:
If you need to convert the code to uppercase, one pattern you could follow is create a static `create()` method that transforms the value. Then you don't need two fields.

```
internal data class CountryCode(
    @Size(2) val value: String,
) {
  companion object {
    fun create(value: String) = CountryCode(value.toUpperCase(Locale.ROOT))
  }
}
```